src/data/downloads.tsx

```diff
@@ -57,36 +57,8 @@ export const Downloads: Array<Download> = [
             <pre>
               <code>{`curl https://repo.jellyfin.org/install-debuntu.sh | sudo bash`}</code>
             </pre>
-            <p>
-              If you do not have <code>curl</code> installed, you can try <code>wget</code> instead:
-            </p>
-            <pre>
-              <code>{`wget -O- https://repo.jellyfin.org/install-debuntu.sh | sudo bash`}</code>
-            </pre>
-            <p>For more advanced users, the full steps are:</p>
-            <pre>
-              <code>
-                {`sudo apt install curl gnupg
-sudo mkdir /etc/apt/keyrings
-DISTRO="$( awk -F'=' '/^ID=/{ print $NF }' /etc/os-release )"
-CODENAME="$( awk -F'=' '/^VERSION_CODENAME=/{ print $NF }' /etc/os-release )"
-curl -fsSL https://repo.jellyfin.org/\${DISTRO}/jellyfin_team.gpg.key | sudo gpg --dearmor -o /etc/apt/keyrings/jellyfin.gpg
-cat <<EOF | sudo tee /etc/apt/sources.list.d/jellyfin.sources
-Types: deb
-URIs: https://repo.jellyfin.org/\${DISTRO}
-Suites: \${CODENAME}
-Components: main
-Architectures: $( dpkg --print-architecture )
-Signed-By: /etc/apt/keyrings/jellyfin.gpg
-EOF
-sudo apt update
-sudo apt install jellyfin`}
-              </code>
-            </pre>
-            <p>
-              <b>Note:</b>: If you are running a non-Debian, non-Ubuntu derivative, ensure the <code>$DISTRO</code> and{' '}
-              <code>$CODENAME</code> are valid Debian or Ubuntu values!
-            </p>
+            <p>If you do not have <code>curl</code> installed, you can use <code>wget -O-</code> instead of <code>curl</code>.</p>
+            <p>For more advanced users, the full steps can be [found in the docs](https://jellyfin.org/docs/general/installation/linux#debuntu).</p>
```

Comment:
I'm iffy on this link. I presume it would be correct but I'm not exactly sure how well Docusaurus handles this.

Reply:
You can do relative redirects too afaik

Reply:
Oh it's more the # part, given that the actual section has a longer title :D